setup.py

```diff
@@ -52,7 +52,6 @@ def run(self):
 
 setup(
     name="RIMEz",
-    version="0.1.0",
```

Comment:
I know that we're using `setuptools_scm`, but I think if there are no tagged releases, it falls back on the version defined here and then adds git info onto the end) I think if we get rid of this, we'll have to make tagged releases on github (which is fine, just something to keep in mind).

Reply:
Hmmm, I see. I don't have a ton of experience with `scm` either. This may come back to the issue raised in #17 . However, given that this setup seems to "work", I'm all for just making a git tag on this merge commit something like `v0.1.1`?

Reply:
Sure, that works for me.